### PR TITLE
fix typo in json viewmodel

### PR DESCRIPTION
**Pull request by superyyrrzz**
fix https://github.com/dotnet/docfx/issues/434

@vwxyzh @ansyral @qinezh @hellosnow @chenkennt

### CODE REVIEW

**Comment by qinezh** [comment]
:+1:

File: src/Microsoft.DocAsCode.Build.ManagedReference/BuildOutputs/ApiNames.cs [hunk 1]
```diff
@@ -24,8 +24,8 @@ public class ApiNames
         [JsonProperty("name")]
         public List<ApiLanguageValuePair> Name { get; set; }
 
-        [YamlMember(Alias = "nameWithTpye")]
-        [JsonProperty("nameWithTpye")]
+        [YamlMember(Alias = "nameWithType")]
+        [JsonProperty("nameWithType")]
         public List<ApiLanguageValuePair> NameWithTpye { get; set; }
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Build.ManagedReference/BuildOutputs/ApiNames.cs]: also update this typo

**Comment by hellosnow** [comment]
:shipit:

**Comment by vwxyzh** [comment]
👍

**Comment by ansyral** [comment]
:+1:

**Pull request merged**